web_frontend/config/urls.py

```diff
@@ -12,3 +14,7 @@
     url(r'^job_progress/', include('osmaxx.job_progress.urls', namespace='job_progress')),
     url(r'^pages/', include('osmaxx.core.urls', namespace='pages')),
 ]
+
+if settings.DEBUG:
+    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT, show_indexes=True) +\
+        static(settings.STATIC_URL, document_root=settings.STATIC_ROOT, show_indexes=True)
```

Comment:
`static()` returns an empty collection unless in debug mode, so you can unconditionally append it.